Cargo.toml

```
@@ -74,4 +74,9 @@ alloy-rpc-types-engine = { version = "0.3.0", features = [
     "ssz",
 ] }
 alloy-rpc-types-eth = { version = "0.3.0" }
-alloy-signer-local = { version = "0.3.0" }
\ No newline at end of file
+alloy-signer-local = { version = "0.3.0" }
+
+[patch.crates-io]
+alloy-eips = { git = "https://github.com/alloy-rs/alloy", branch = "rkrasiuk/bid-submission-v4" }
+alloy-rpc-types-beacon = { git = "https://github.com/alloy-rs/alloy", branch = "rkrasiuk/bid-submission-v4" }
+alloy-rpc-types-engine = { git = "https://github.com/alloy-rs/alloy", branch = "rkrasiuk/bid-submission-v4" }
```

These can all be `version = "0.3.6"` now?

https://github.com/alloy-rs/alloy/releases/tag/v0.3.6

```suggestion
alloy-eips = { version = "0.3.6" }
alloy-rpc-types-beacon = { version = "0.3.6" }
alloy-rpc-types-engine = { version = "0.3.6" }
```

---

@ryanschneider the spec is being changed, so i'm holding off moving it from draft https://github.com/ethereum/execution-apis/pull/587